### DISCUSSION TITLE
Scrum Prep Milestone 3-4

### DISCUSSION THREAD

**Discussion by NatiPerez459** [uprm-inso4101-2024-2025-s1/semester-project-mercado-colegial-online#228] [General]
Here you will post what you have worked on during this past week.

**Comment by Cabinetman** [comment]
This past week, I’ve been working on creating a Petri Net for the website's structural interactions.

**Comment by CovasJJ** [comment]
This week I've been working on adding Stakeholder Information to the documentation.

**Comment by cristian-barreras** [comment]
I have been working on the Petri Net for the website's structural interactions and documentation this week.

**Comment by JadStelar** [comment]
Wasn't able to do much this week, will be working on and creating tasks during the weekend

**Comment by JuanDelPueblo** [comment]
This week I've created and began work on three tasks adding new sections to the documentation: Stakeholder Information, Business Processes, and Domain Facets.

**Comment by RyanVelez101** [comment]
This week I’ve been working on updating the cart webpage UI

**Comment by CactusByte** [comment]
I'll be working on the frontend responsive components this week

**Comment by PerezMarcelo** [comment]
Plan on working on documentation tasks.

**Comment by Kaariinaa08** [comment]
Created and worked on the Cost Analysis and Fault Analysis sub-sections of the Milestone #3 Documentation.
- Karina Lopez Rodriguez

**Comment by Christian2101** [comment]
Continued to fix the documentation using the professors feedback

**Comment by elliot-gomez** [comment]
This week, I created the task for documenting the domain requirements, focusing on deriving precise and testable requirements based on the domain properties. I’ve started working on this task by addressing milestone 2 feedback and identifying some initial requirements.
- Elliot J. Gomez Coriano

**Comment by AlvinBadillo** [comment]
This week I started working on connecting the sell model to the database, I found that there was an issue with the sell controller so I will create a new task to fix that issue.

**Comment by CdoxHype** [comment]
I have been working on Home Page, Nav Bar and Footer enhancement.

-Julio Camacho

**Comment by YampierFernandez** [comment]
Worked on lecture topic task for Risck tree diagrams

**Comment by gcal77** [comment]
This week i've been working on adding more risk trees to the documentation. As well as starting to work on other lecture topic tasks.

-Gabriel Calvente

**Comment by franrov4** [comment]
Worked on a Lecture Topic task regarding a hypothetical cost analysis for the product.
